### ISSUE TITLE
Error validating rss feed

### ISSUE THREAD

**Issue by thiagoalmeidasa** [TryGhost/Wave#12]
First of all, great work on this theme, it is amazing and I'm looking forward to using it on my podcast/blog.

But I tried to validate its RSS feed against some known tools and it failed, making me wonder if it will work when I try to import it to Spotify or iTunes.

https://castfeedvalidator.com/?url=https://wave.ghost.io/rss/

```
ERROR: The link tag with attribute rel='self' does not point to a valid RSS feed and will therefore be denied by Apple and Google.
WARNING: Your feed has no language tag. Apple Podcasts will not accept a feed with no language specified.
WARNING: Found 10 episodes with an invalid length. The length should be a nonzero value specified in the enclosure tag, and refers to the file length in bytes.
WARNING: Found 8 items with an enclosure URL that has already been used in your feed. These episodes will be ignored by Apple Podcasts.
ERROR: No itunes:image tag in feed. Cannot validate feed artwork.
FATAL: No artwork URL found. Did not finish artwork tests.
WARNING: Your title ID3 tag is missing.
WARNING: Your bitrate mode is not set to CBR. This is the optimal setting for podcast audio.
```

**Comment by minimaluminium** [comment]
Hey @thiagoalmeidasa,

Went through the validation warnings and errors.

> ERROR: The link tag with attribute rel='self' does not point to a valid RSS feed

Fixed this one in https://github.com/TryGhost/Wave/commit/e7c7bbb431f3c3d439fd2e55041e67d3a1f15c94.

> WARNING: Your feed has no language tag.

The feed has a language tag, but the demo site hasn't been updated to Ghost 4.x yet, and that's the reason why `{{@site.locale}}` not working on the demo.

> WARNING: Found 10 episodes with an invalid length.

Calculating file length on the fly would be impossible for how we're generating the feed unless we use the `og_description` field for full info like: `url="https://example.com/episode-1.mp3" length="1500" type="audio/mp3"`. Still not sure about this 🤔 

> WARNING: Found 8 items with an enclosure URL that has already been used in your feed.

That's because we used same audio URL for demo purpose.

> ERROR: No itunes:image tag in feed. Cannot validate feed artwork.
FATAL: No artwork URL found. Did not finish artwork tests.

Site publication icon is used as a feed artwork [here](https://github.com/TryGhost/Wave/blob/e7c7bbb431f3c3d439fd2e55041e67d3a1f15c94/rss.hbs#L28). On the demo site, we don't have any publication icon selected. When you select an image as an icon, it'll be validated with no issue.

> WARNING: Your title ID3 tag is missing.
WARNING: Your bitrate mode is not set to CBR. This is the optimal setting for podcast audio.

These are related with the audio files we used on the demo.
